mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -61,12 +62,13 @@ class ConvConverter final
 public:
   using OpConversionPattern<migraphx::ConvolutionOp>::OpConversionPattern;
 
-  Value getZero(Location loc, Type elemType,
-                ConversionPatternRewriter &rewriter) const {
-    auto biasTy = RankedTensorType::get({1}, elemType);
-    auto zeroAttr =
-        DenseElementsAttr::get(biasTy, rewriter.getZeroAttr(elemType));
-    return rewriter.create<arith::ConstantOp>(loc, zeroAttr);
+  Value getZeroBias(Location loc, Type elemType, int64_t filterOutputChannels,
```

Comment:
Is this change necessary? this is an WA to seal the gap between MIGraphX and tosa conv2d where MIGraphX doesn't get bias as an argument but always converts to broadcast_bias+bias tensor.
I'd like to have it as a minimal code in IR.

Reply:
The current TOSA that is being created only works iff if its lowered to Rock dialect. i.e. migraphx to tosa to linalg flow is otherwise broken. (Not sure we care about that)

Alternatively we need to fix the TosaToLinalg but I feel we should do that upstream directly.

In conclusion, its not needed for the rock path as of today but it feels like we are creating TOSA that would work only for our flow. WDYT?

Reply:
I see, 
I thought tosa.conv2d allows 1-element bias but maybe not, or just TosaToLinalg as you commented.
Just checked the spec and it says 'per output channel', so this makes better sense.